tests/app/user/test_rest.py

```diff
@@ -372,6 +372,7 @@ def test_post_user_attribute_with_updated_by(
     assert json_resp['data'][user_attribute] == user_value
 
     if arguments:
+        assert mock_persist_notification.call_count == 1
```

Comment:
Was 2 calls before making the change. One "something on your profile changed" and another "an admin changed a thing". Making sure only the "admin changed a thing" is sent